internal/datastore/memdb/caveat.go

```diff
@@ -0,0 +1,118 @@
+package memdb
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/authzed/spicedb/pkg/datastore"
+	core "github.com/authzed/spicedb/pkg/proto/core/v1"
+
+	"github.com/hashicorp/go-memdb"
+)
+
+const tableCaveats = "caveats"
+
+type caveat struct {
+	id         datastore.CaveatID
+	name       string
+	expression []byte
+}
+
+func (c *caveat) CoreCaveat() *core.Caveat {
```

Comment:
`CoreCaveat` is a big vague of a name... maybe `Unwrap()`?

Reply:
I followed the same convention used for `relationship` which has a method to create create a `v1.Relationship` named `Relationship`. I'm OK with `Unwrap` 👍🏻